test/runtests.jl

```diff
@@ -9,6 +9,7 @@ include("tabulated_tests.jl")
 unit_tests = [
 UnitTest("DataLayouts fill"                        ,"DataLayouts/unit_fill.jl"),
 UnitTest("DataLayouts ndims"                       ,"DataLayouts/unit_ndims.jl"),
+UnitTest("DataLayouts ndims"                       ,"DataLayouts/has_uniform_datalayouts.jl"),
```

Comment:
The title here was not updated:
```suggestion
UnitTest("DataLayouts uniform datalayouts"              ,"DataLayouts/has_uniform_datalayouts.jl"),
```

Reply:
Ah, yes, thanks!